### ISSUE TITLE
Object Spread: "Warning: semicolon - Missing semicolon"

### ISSUE THREAD

**Issue by nickfujita** [TypeStrong/atom-typescript#1156]
When using objects and functions that return objects in a object spread, I am getting a warning that a semicolon is missing. I encountered this with the default config with the sample code below:

```
let firstObj = {
  a: 1,
  b: 2,
};

let secongObj = {
  b: 3,
  c: 4,
};

let combinedObj = {
  d: 5,             //Warning: semicolon - Missing semicolon
  ...firstObj,      //Warning: semicolon - Missing semicolon
  ...secondObj,     //Warning: semicolon - Missing semicolon
  ...getThirdObj(), //Warning: semicolon - Missing semicolon
}

function getThirdObj() {
  return {
    d: 6,
    e: 7,
  };
}
```

Is there a valid use case where this should be warning of a missing semicolon when using the spread on `combinedObj`?

**Comment by nickfujita** [comment]
Closing, as this is a problem with `linter-tslint` rather than `atom-typescript`.
